ui/src/app/job-details/panels/panels.component.ts

```diff
@@ -16,13 +16,18 @@ import {ResourceUtils} from '../../shared/utils/resource-utils';
   styleUrls: ['./panels.component.css'],
 })
 export class JobPanelsComponent implements OnChanges {
+  // Filter out 'envs' and 'logs' from extendedFields since these are maps and
+  // require special treatment for display.
+  private readonly extensionsWhiteList: string[] = ['userId', 'statusDetail', 'lastUpdate', 'parentJobId'];
```

Comment:
I'd do one of two things here:

- Use this array to inform the order on the details page
- Use a Set instead

Reply:
Done, also cleaned up a few small things.